src/main/kotlin/org/octopusden/octopus/automation/teamcity/TeamcityCreateBuildChainCommand.kt

```diff
@@ -181,11 +188,15 @@ class TeamcityCreateBuildChainCommand : CliktCommand(name = COMMAND) {
         )
     }
 
-    private fun setBuildTypeParameter(buildTypeId: String, name: String, value: String) =
+    private fun setBuildTypeParameter(buildTypeId: String, name: String, value: String) {
         client.setParameter(ConfigurationType.BUILD_TYPE, buildTypeId, name, value)
+        log.info("Set parameter $name value $value for build configuration with id $buildTypeId")
```

Comment:
Simple function. it's Kotlin, use also:
`
client.setParameter(ConfigurationType.BUILD_TYPE, buildTypeId, name, value).also { log.info ....}
`